CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ accidentally triggering the load of a previous DB version.**
 ## Unreleased
 
 **Bugfixes**
+* #4804 Fix timestamp out of range
```

Comment:
In the light of our recent discussion about changelog messages, we could make it more readable for users, something along the lines of the PR description perhaps?

Reply:
Good point, thanks.